.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -30,8 +32,8 @@ Contents of `hie.yaml`:
 <!-- Tell us what happens instead. -->
 
 ### Include debug information
-Execute in the root of your project the command `haskell-language-server --debug .` and paste the logs here:
-
+Execute in the root of your project the command `haskell-language-server-wrapper --debug .` and paste the logs here:
+(if you are using the vscode extension check the executable location [here](https://github.com/haskell/vscode-haskell#downloaded-binaries))
```

Comment:
I wonder if we should just write more about this on the troubleshooting page, and then here we could say "If you don't know where to find the `haskell-language-server` exectuable, consult the troubleshooting page"

Reply:
Yeah, although a direct reference to the location may be helpful too? I mean, we are linking the troubleshooting page in the beginning but in this point the focus is locate it and the table linked gives you the path